test/test-util/src/kafka/kafka_client.rs

```diff
@@ -30,6 +30,7 @@ impl KafkaClient {
         let mut config = ClientConfig::new();
         config.set("bootstrap.servers", kafka_url);
         config.set("group.id", group_id);
+        config.set("enable.idempotence", "true");
```

Comment:
this adds it to other uses of KafkaClient which includes billing demo right? could we change the api here so that it adds a Option<&[(String, String)]> or something like that so that users can change configs arbitrarily. I think what I'd like to do for billing-demo and other future demos is to have a config command line arg so you could pass in like --producer-config "enable.idempotence:true,foo:bar, ..."

Reply:
Sounds good! Do you think `enable.idempotence` should be added to the billing demo now?

Reply:
i think its fine to leave it as is for now. thanks for checking though!